src/services/storage/drivers/relational/migrations/20210210214139_urlsInformation.ts

```diff
@@ -0,0 +1,24 @@
+import * as Knex from 'knex'
+
+export async function up(knex: Knex): Promise<void> {
+	await knex.raw('PRAGMA foreign_keys = ON')
```

Comment:
What's PRAGMA?

![image](https://user-images.githubusercontent.com/26040325/109775765-92d6cd00-7c0a-11eb-9bff-ce1264302711.png)

Reply:
https://stackoverflow.com/a/59813897/4866465

Reply:
Are you sure it works in Postgres and doesn't throw an exception? the issue is based on SQLite

Reply:
you are correct. Fixed